client/src/main/java/com/sap/mobile/services/client/XsuaaAuthorizationRequestInterceptor.java

```diff
@@ -59,13 +52,4 @@ public ClientHttpResponse intercept(final HttpRequest request, final byte[] body
 		return execution.execute(request, body);
 	}
 
-	private boolean isMissingTenantError(final Exception e) {
-		final Throwable cause = e.getCause();
-		if (cause instanceof OAuth2ServiceException) {
-			final Integer statusCode = ((OAuth2ServiceException) cause).getHttpStatusCode();
-			return statusCode != null && statusCode.equals(HttpStatus.NOT_FOUND.value());
-		}
-
-		return false;
-	}
```

Comment:
This actually changed the behavior of the library.
I've created https://github.com/SAP/cloud-security-services-integration-library/pull/1251 to restore the `getHttpStatusCode` method. Let's wait for them to accept and release the change and then we can keep this logic.